gadget/install/install.go

```diff
@@ -525,6 +525,25 @@ func WriteContent(onVolumes map[string]*gadget.Volume, allLaidOutVols map[string
 	return onDiskVols, nil
 }
 
+// getMntPointForPart tells us where to mount a given structure so we
+// match what the functions that write something expect.
+func getMntPointForPart(part *gadget.VolumeStructure) (mntPt string) {
+	switch part.Role {
+	case gadget.SystemSeed, gadget.SystemSeedNull:
+		mntPt = boot.InitramfsUbuntuSeedDir
+	case gadget.SystemBoot:
+		mntPt = boot.InitramfsUbuntuBootDir
+	case gadget.SystemSave:
+		mntPt = boot.InitramfsUbuntuSaveDir
+	case gadget.SystemData:
+		mntPt = boot.InstallUbuntuDataDir
+	default:
+		mntPt = filepath.Join(boot.InitramfsRunMntDir, part.Name)
+	}
+
```

Comment:
nitpick: not sure we need the blank line here

Reply:
Removed